docs/stencil-docs/customizing-checkout/checkout-sdk-example.md

```diff
@@ -116,421 +116,24 @@ lineNumbers: true
-
-# add directory to hold example code
-mkdir assets/js/checkout-app
-
-# copy example source code to directory you just created
-cp -R ../checkout-sdk-example/src/* assets/js/checkout-app/
-```
-
-### Import Dependencies
-
-In `/assets/js/app.js`, add the following lines below the existing `import` statements:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```js
-import React from 'react';
-import ReactDOM from 'react-dom';
-import Checkout from './checkout-app/Checkout/checkout';
-```
-
-### Define a Function to Initialize React
-
-In `/assets/js/app.js`, add the following lines to the end:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```js
-window.initReact = function initReact() {
-    ReactDOM.render(
-        React.createElement(Checkout, null, null),
-        document.getElementById('checkout-app')
-    );
-};
-```
-
-### Customize the Checkout Template
-
-In `/templates/pages/checkout.html`, add the following code between `{{{ footer.scripts }}}` and `{{/partial}}`:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```html
-<script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
-
-<script type="text/javascript" defer>
-    // Exported in app.js
-    window.initReact();
-</script>
-```
-
-Then, in `/templates/pages/checkout.html`, replace:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```html
-{{ checkout.checkout_content }}
-```
-
-with:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name"></div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: ""
-subtitle: ""
-lineNumbers: true
--->
-
-```
-<div id="checkout-app"></div>
-```
+1. Navigate to the [Open source checkout](https://github.com/bigcommerce/checkout-js)repository. 
+2. Clone the respository.
+3. Move into the checkout-js directory.
+5. Run `npm ci` to install dependencies.
+6. Build out your custom checkout page by making changes to the source code located in the checkout.js folder. 
+7. Run `npm run build` to build the source code.
 
 ### Congratulations!

```

Comment:
this is no longer relevant, as `stencil start` is specific for theme integration..